content/rc/security/multi-factor-authentication.md

```diff
@@ -24,10 +24,12 @@ To configure MFA for your user account:
 
 1. Log in to your account.
 2. From the Redis Enterprise Cloud menu, click on your name to enter the **User Profile** view.
-3. Under your user profile, click **Multi-Factor Authentication**.
-4. From the **Multi-Factor Authentication** view, click **Activate Now**.
-5. Enter your mobile phone number.
-6. You will receive a confirmation code sent by text message. Enter the code and click **Verify**.
+3. Under your user profile, locate **Multi-factor authentication**.
+
+    {{<image filename="images/rc/multi-factor-authentication-user-profile.png" alt="Multi-factor authentication is located on the user profile page." >}}{{< /image >}}
+
+4. Enter your mobile phone number, and click **Send code**.
+5. You will receive a confirmation code sent by text message. Enter the code and click **Verify**.
```

Comment:
Does the admin console show a prompt where you are supposed to enter the code? If so, you might want to clarify that here.
```suggestion
5. You will receive a confirmation code sent by text message. Enter the code and select **Verify**.
```

Reply:
@Sawdustie, I changed this back to unresolved because I'm not sure you saw my question about clarifying that you enter the code into a prompt in the admin console (if this statement is true).

Reply:
You're right I did not see it, sorry. The admin console does show a prompt where the code should be entered, I will add that.